packages/core/test/financial-templates/common/financial-product-libraries/long-short-pair-libraries/RangeBondLongShortPairFinancialProductLibrary.js

```diff
@@ -34,35 +34,20 @@ contract("RangeBondLongShortPairFinancialProductLibrary", function () {
       assert.equal(setParams.highPriceRange.toString(), highPriceRange);
     });
     it("Can not re-use existing LSP contract address", async () => {
-      await rangeBondLSPFPL.setLongShortPairParameters(
-        lspMock.address,
-
-        highPriceRange,
-        lowPriceRange
-      );
+      await rangeBondLSPFPL.setLongShortPairParameters(lspMock.address, highPriceRange, lowPriceRange);
 
       // Second attempt should revert.
       assert(
         await didContractThrow(
-          rangeBondLSPFPL.setLongShortPairParameters(
```

Comment:
this test was generating false positives! the parameterization of `lowPriceRange` first then `highPriceRange` will revert due to the other conditions that validate the correctness of the parameters, not the overriding of parameters.